pkg/vsphere/disk/disk_manager.go

```diff
@@ -421,3 +422,75 @@ func (m *Manager) devicePathByURI(op trace.Operation, datastoreURI *object.Datas
 
 	return sysPath, nil
 }
+
+// MountChildAndAncestor attaches and mounts disks representing a child vmdk and its ancestor.
+// It returns their mount paths if successful, empty strings and error otherwise.
+func (m *Manager) MountChildAndAncestor(op trace.Operation, child, ancestor *object.DatastorePath) (string, string, error) {
```

Comment:
There's a good amount of boilerplate for attaching/mounting disks (2 attaches, 2 mounts, with all the implied `defer` calls, etc) that was originally in place in both the image and volume stores, so I was trying this as a way to cut down on the DRY. But I agree, I don't really like it like this and already plan on fixing it up.